src/Plugin.php

```diff
@@ -55,6 +73,9 @@ public function __construct(array $config = [])
             throw new \InvalidArgumentException("Missing required configuration key 'nicks'");
         }
         $this->setNicks($config['nicks']);
+        if (!empty($config['recovery'])) {
```

Comment:
I think this conditional will evaluate to `true` if `$config['recovery']` is explicitly set to `false`?